src/classes/AppState.js

```diff
@@ -5,50 +5,62 @@ export class AppState {
     this.is_auth = false;
     this.roles = [];
   }
-
+  
   getUserID() {
     return this.user._id;
   }
-
-  getUser(uid) {
+  
+  async getUser(uid) {
```

Comment:
This doesn't make sense to use async and await here, since it already returns a promise.